### DISCUSSION TITLE
Nordpool electricity price panel in Current Charge dashboard

### DISCUSSION THREAD

**Discussion by stauffenberg2020** [jheredianet/Teslamate-CustomGrafanaDashboards#82] [General]
Hi @jheredianet,

I was wondering if it would be somehow possible to display the instantaneous electricity price (at least for the current day) from Nordpool/others in a new panel below all the current panels in the Current Charge View dashboard. I am aware that this data is not written into the teslamate database, But, would it be possible to directly scrape using standard Grafana options from Nordpool or Energinet or any other website and only display it (No need to save into the database).

**Comment by jheredianet** [comment]
HI @stauffenberg2020 Although it is a good idea, when it comes to Teslamate we have to think globally and not just on a specific country/region since otherwise it would not be useful at all, only for someones. 

I also thought about it when wrote that dashboard because where I live the price of electricity varies daily every hour, but I had to give up on the idea because it was not going to be useful for everyone (even for users who live around me). Everyone has their own use cases and specific contracts. 

In any case, I think the most useful thing is that at the end of the charge, it does the calculation and automatically updates the cost in the database and this way you always have it up to date and avoid having to enter it manually. I have it implemented that way using NodeRed (using MQTT), when the charge is finished, I have a script that reads the price from my electricity company in the period in which the car charged and then updates the "Charging Process" table with the corresponding total cost.
